autograd/nn.py

```diff
@@ -246,6 +246,44 @@ def forward(self, x):
         return pooled
 
 
+class ResidualBlock(Module):
```

Comment:
Just moving it from `resnet.py` to here to consolidate all neural network modules.